src/algebra/absolute_value.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2021 Anne Baanen. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro, Anne Baanen
 -/
-import algebra.ordered_field
+import algebra.order.field
 
```

Comment:
What do you think about moving `absolute_value.lean` and `euclidean_absolute_value.lean` to `algebra/order` as well?

Reply:
I think `algebra.module.ordered` could also become `algebra.order.module`.

Reply:
And off `algebra.module.pi` could be split `algebra.order.pi`.

Reply:
There isn't anything about orders in `algebra.module.pi`. Regarding `algebra.module.ordered`, I'm hesitant to move it because it isn't about Type-valued typeclasses.

Reply:
Oh yes, forget me about `algebra.module.pi`, I got confused.

What about having `absolute_value` and `euclidean_absolute_value` be `algebra.order.absolute_value.basic` and `algebra.order.absolute_value.euclidean`?

Reply:
I think that's out of scope. I just want to restructure the order files out of the algebra hierarchy. We can follow up with that if you wish. My goal with this reorganization is to make it simpler to build the algebra+order hierarchy, so abv doesn't really fit into my foreseen scope.